### PR TITLE
package.json update - gov-frontend 4.7.0

### PR DESCRIPTION
**Pull request by olusegz07**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/EXUI-1219

### Change description ###
* Added updated common-lib 2.0.12 which includes govuk-frontend 4.7.0 and its related fixes 
* Updated govuk-frontend to 4.7.0 
* test fixes

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by HYadavHMCTS — DISMISSED** [review]

File: test_codecept/integration/tests/get_Organisation_User_Details.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ suite('Manage Org -> Get Organisation User details', function() {
       response.status.should.be.eql(200);
       console.log(response.data.users[0].email);
       // response.data.users[14].userIdentifier.should.be.eql('fb25e17b-2456-4ce0-909d-071b68d10d59');
-      response.data.users[0].firstName.should.be.eql('Vamshi');
-      response.data.users[0].lastName.should.be.eql('Muniganti');
+      response.data.users[0].firstName.should.be.eql('Jason');
```
Comment (andywilkinshmcts) [inline test_codecept/integration/tests/get_Organisation_User_Details.ts:12]: This is just going to break again. I would change this to search the users array for specific user.

File: test_codecept/integration/tests/get_Organisation_User_Details.ts [hunk 2]
```diff
@@ -9,9 +9,9 @@ suite('Manage Org -> Get Organisation User details', function() {
...
+      response.data.users[0].lastName.should.be.eql('Lee');
       // response.data.users[0].email.should.be.eql('xuiapitestuser@mailnesia.com');
-      response.data.users[0].idamStatus.should.be.eql('PENDING');
```
Comment (andywilkinshmcts) [inline test_codecept/integration/tests/get_Organisation_User_Details.ts:15]: Again - search for a specific user and then check that that user's status is as expected

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -79,7 +79,7 @@
     "@hmcts/media-viewer": "2.10.1-RC.1",
     "@hmcts/nodejs-healthcheck": "1.7.0",
     "@hmcts/properties-volume": "0.0.13",
-    "@hmcts/rpx-xui-common-lib": "2.0.10",
+    "@hmcts/rpx-xui-common-lib": "2.0.16",
```
Comment (shaed-parkar) [inline package.json:82]: `"govuk-frontend": "3.14.0",` also needs to be upgraded else the project will conflict and I believe use v3 instead

**Review by RiteshHMCTS — APPROVED** [review]

**Pull request merged**